### ISSUE TITLE
Add alt text for images and font icons in uberbar in manager (accessibility defect)

### ISSUE THREAD

**Issue by paulbohman** [modxcms/revolution#12152]
For compliance with Web Content Accessibility Guidelines (WCAG)

The gravatar image has no alt text
The gear icon and help icon need text. Options include:
Use visually hidden text that only screenr readers can access (use CSS clip or offscreen text) and hide the icon itself in case the screen reader tries to read it (it will almost surely read it incorrectly):
<pre>
&lt;span class=&quot;icon-gear icon icon-large&quot; aria-hidden=&quot;true&quot;&gt;&lt;/span&gt;&lt;span class=&quot;visually-hidden&quot;&gt;System Menu&lt;/span&gt;</pre>

Or use role=&quot;image&quot; and aria-label:

<pre>&lt;span class=&quot;icon-gear icon icon-large&quot; role=&quot;image&quot; aria-label=&quot;System Menu&quot;&gt;&lt;/span&gt;</pre>

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#uberbar

**Comment by rthrash** [comment]
This is being addressed at the [MODX Accessible CMS Project](https://github.com/modxcms/a11y/issues/), and with Paul's agreement has been moved there. For more details, see https://github.com/modxcms/revolution/issues/12156#issuecomment-72382480. Thank you!
